wallet-new/src/Cardano/Wallet/API/V1/Types.hs

```diff
@@ -214,8 +216,12 @@ instance Bounded a => Bounded (V1 a) where
 instance Buildable a => Buildable (V1 a) where
     build (V1 x) = bprint build x
 
+instance Buildable a => Buildable [V1 a] where
+    build = bprint listJson
+
 instance Buildable (SecureLog a) => Buildable (SecureLog (V1 a)) where
-    build (SecureLog vx) = bprint build (SecureLog vx)
+    build = bprint build
```

Comment:
Let me resurrect for a while and note, that your rewrite reveals me writing a nonsense here.
With same grade of success I could write `build = build` :smile:

Reply:
I wonder how it even compiles actually. Could it be some `UndecidableInstances` magic of something that will blow up at runtime :o ?

Reply:
It will probably just loop.